tracee-rules/benchmark/helpers_test.go

```diff
@@ -0,0 +1,113 @@
+package benchmark
+
+import (
+	_ "embed"
+	"encoding/gob"
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+	"testing"
+
+	tracee "github.com/aquasecurity/tracee/tracee-ebpf/tracee/external"
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+)
+
+var (
+	innocentEvent = tracee.Event{
```

Comment:
would be better to use a full event, with data in the fields, so that the load is more realistic (same goes for other events)

Reply:
Added values for fields based on real events captured by tracee-ebpf